tox.ini

```diff
@@ -1,2 +1,17 @@
+[tox]
+envlist = py27, py36
+skip_install = True
+skipsdist = True
+
+[testenv]
+deps = pipenv
+commands =
+    python --version
+    pipenv install --dev --skip-lock
+    pytest
+
+
```

Comment:
The changes regarding tox are not mentioned in the commit message. Are they stongly related to the casting? If not I would put them into a separate commit.

Reply:
Ich habe es gesplitted.